src/Item/CreateItem/CreateItem.js

```diff
@@ -0,0 +1,118 @@
+import React, {
+  useState,
+  useCallback,
+  useEffect,
+  useMemo,
+} from 'react';
+import PropTypes from 'prop-types';
+import {
+  useHistory,
+  useLocation,
+} from 'react-router-dom';
+
+import {
+  stripesConnect,
+  stripesShape,
+} from '@folio/stripes/core';
+import {
+  LoadingView,
+} from '@folio/stripes/components';
+
+import ItemForm from '../../edit/items/ItemForm';
+
+const CreateItem = ({
+  stripes,
+  referenceData,
+  mutator,
+  instanceId,
+  holdingId,
+}) => {
+  const history = useHistory();
```

Comment:
will it work on testing, where we don't have react-router v5?

Reply:
good question, seems no

@zburke are there any plans to bump react-router version in stripes?

Reply:
We've had plans to bump `react-router` to `v5` for ages now. [STRIPES-674](https://issues.folio.org/browse/STRIPES-674) captures the work. I started this back in September and got 90% of the way there, including PRs in all platform-core apps, before getting hung up on [STCOR-390](https://issues.folio.org/browse/STCOR-390) and needing to drop it and move on to other things. I still feel badly about that 😞